.gitmodules

```diff
@@ -1,8 +1,8 @@
 [submodule "config"]
 	path = config
-	url = https://github.com/georchestra/datadir.git
+	url = git@github.com:georchestra/datadir.git
 	branch = docker-master
 [submodule "resources/static"]
 	path = resources/static
-	url = https://github.com/georchestra/htdocs.git
+	url = git@github.com:georchestra/htdocs.git
```

Comment:
We should keep https here, as some firewalls are really tough with git.

Reply:
https is okay for none auth clone but not for everything it does no works (auth, modifications, pushing ...)